qiskit_nature/components/variational_forms/chc.py

```diff
@@ -20,8 +20,7 @@
 
 from qiskit.circuit import ParameterVector, Parameter
 
-from qiskit.aqua.components.variational_forms import VariationalForm
-from qiskit.aqua.components.initial_states import InitialState
+from qiskit.algorithms.variational_forms import VariationalForm
```

Comment:
Why does this class exist in Terra? This should *not* be the case and instead we should refactor all classes in Nature which depend on it to become a `BluePrintCircuit`.

@Cryoris tagging to verify :+1:

Reply:
My guess it was maintained there pending its complete removal from Qiskit, VQE still supports the present UCCSD based on VariationalForm. I think once the UCCSD refactor is done we should eliminate this from Terra. If we don't mind breaking things in advance, then of course it could be removed from Terra now.

Reply:
I think we should remove it from Terra right away. All imports of the `VaritionalForm` class are based on the Aqua module.